### PR TITLE
header added fonctionnal

### PR DESCRIPTION
**Pull request by LuckyShuii**

### CODE REVIEW

**Review by Richarddesnoues — APPROVED** [review]

File: public/assets/js/header.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        // Open or close the burger menue
+        showResponsiveMenu()
+    })
+
+    mediaQueryChecker() // Check if the window size is changing
+}
+
+// Open or close the burger menue, called in onload()
+function showResponsiveMenu() {
+    if (!burgerIcone.classList.contains('opened')) {
```
[the remark this answers is not in the document]
Comment (Benja-san) [reply public/assets/js/header.js:22]: toggle function

File: public/assets/js/header.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+let logoHeader = document.querySelector("#logo_header")
+let myHeader = document.querySelector("header")
+let myFooter = document.querySelector("footer")
+let myMain = document.querySelector("main")
+let burgerIcone = document.querySelector("#burger_icon")
```
Comment (Benja-san) [inline public/assets/js/header.js:5]: Icon * ?

File: public/assets/js/header.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+let burgerMenue = document.querySelector("#burger_menue")
```
Comment (Benja-san) [inline public/assets/js/header.js:6]: Menu * ?

File: src/View/components/_header.html.twig [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <button id="burger_icon" class="menuBurgerGlobal" aria-label="Main Menu">
+        <svg width="50" height="50" viewBox="0 0 100 100">
+            <path class="line line1" d="M 20,29.000046 H 80.000231 C 80.000231,29.000046 94.498839,28.817352 94.532987,66.711331 94.543142,77.980673 90.966081,81.670246 85.259173,81.668997 79.552261,81.667751 75.000211,74.999942 75.000211,74.999942 L 25.000021,25.000058" />
+            <path class="line line2" d="M 20,50 H 80" />
+            <path class="line line3" d="M 20,70.999954 H 80.000231 C 80.000231,70.999954 94.498839,71.182648 94.532987,33.288669 94.543142,22.019327 90.966081,18.329754 85.259173,18.331003 79.552261,18.332249 75.000211,25.000058 75.000211,25.000058 L 25.000021,74.999942" />
+        </svg>
+    </button>
+</header>
+
+<ul id="burger_menue">
```
Comment (Benja-san) [inline src/View/components/_header.html.twig:23]: burger_menu * ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: public/assets/js/header.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
+let logoHeader = document.querySelector("#logo_header")
+let myHeader = document.querySelector("header")
```
[the remark this answers is not in the document]
Comment (LuckyShuii) [reply public/assets/js/header.js:2]: Both variables deleted next commit - they weren't used in the code anyway

File: public/assets/js/header.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+let myFooter = document.querySelector("footer")
```
[the remark this answers is not in the document]
Comment (LuckyShuii) [reply public/assets/js/header.js:3]: changed into questSelectorAll instead to fix this - next commit

File: public/assets/js/header.js (hunk 3 above)
Comment (LuckyShuii) [reply public/assets/js/header.js:6]: fixed - next commit

File: src/View/components/_header.html.twig (hunk 4 above)
Comment (LuckyShuii) [reply src/View/components/_header.html.twig:23]: fixed - next commit

File: public/assets/js/header.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (LuckyShuii) [reply public/assets/js/header.js]: done

**Review by Benja-san — APPROVED** [review]

**Pull request merged**